PukingPredator/Assets/Scripts/Puking.cs

```diff
@@ -25,7 +25,20 @@ public class Puking : InputBehaviour
     private Vector3 pukeDirection => transform.forward + Vector3.up * 0.1f;
 
     /// <summary>
-    /// If pukeForce is greater than this, the puke will also knock back items in front of the player
+    /// The collision tracker used to roughly determine the objects that could
+    /// be affected.
+    /// </summary>
+    [SerializeField]
+    private CollisionTracker pukeExplodeCollisionTracker;
+
+    /// <summary>
+    /// The max distance if you fully charge the puke but had 2 items consumed.
+    /// </summary>
+    private const float PUKE_EXPLODE_BASE_MAX_DISTANCE = 2f;
+
+    /// <summary>
+    /// If pukeForce is greater than this, the puke will also knock back items
+    /// in front of the player.
     /// </summary>
     private const float PUKE_EXPLODE_THRESHOLD = 0.5f;
```

Comment:
Is this being used,

Reply:
the threshold? yes, its used to determine if the explosion happens at all when you puke. i believe the change here was just making the comment 2 lines instead of 1 though